apps/ledger-live-desktop/src/renderer/families/cosmos/RedelegationFlowModal/Body.jsx

```diff
@@ -142,7 +142,7 @@ const Body = ({
 
     const transaction = bridge.updateTransaction(t, {
       mode: "redelegate",
-      validators: [{ address: "", amount: source?.amount ?? BigNumber(0) }],
+      validators: [{ address: validatorDstAddress, amount: source?.amount ?? BigNumber(0) }],
```

Comment:
This seems unrelated to the specific scope of the banner, can you please explain this change?

Reply:
In the actual relegation flow is not been selected by default any destiny provider, in this case follow the story we are using 'ledger' as default redelegator destiny. (Ledger in this case is preselected)